guides/troubleshooting/README.md

```diff
@@ -0,0 +1,72 @@
+# Troubleshooting
+
+While deploying your app or making changes, you may run into a situation where your app will not deploy. This can happen for a number of reasons, and in this guide, you'll learn a few things that may help you get moving again.
+
+## Check the build logs
+
+The platform will pull your app from GitHub and build it when you first create it and anytime you make changes and push them to GitHub. During the build, the platform will run `npm i` and `npm run wpe-build`. It's possible that either for these could fail, and your build logs will tell you what happened.
+
+First, list the builds for an Environment of an App:
+
+First, find your app name by running:
+
+```
+$ wpe alpha apps list
+```
+
+Next, find your Environment ID by running:
+
+```
+$ wpe alpha apps get APP_NAME
+```
+
+Once you have your APP_NAME and ENVIRONMENT_ID, use them in the following command to view your builds:
+
+```
+$ wpe alpha builds list -a APP_NAME -e ENVIRONMENT_ID
+```
+
+This command will show you a list of builds. Now, you can find the build logs for a specific build with the ID of the build using the `get` command:
+
+```
+$ wpe alpha builds get -a APP_NAME -e ENVIRONMENT_ID -b BUILD_ID
+```
+
+Look through the Error Output and Build Output for any issues that may be preventing the build from completing.
+
+## Check `package.json`
```

Comment:
Even though these are about the `package.json` file, I would separate:

## Ensure `package.json` has a `wpe-build` target (or however you want to say that).
and
## Ensure `package.json` lists your dependencies.

Why? Because many people will stop reading this section when they see the bit about dependencies. Making the build target a separate top-level entry is free and calls attention to it.

Reply:
Updated